src/components/cc-email-list/cc-email-list.stories.js

```diff
@@ -7,13 +7,32 @@ const SECONDARY_ADDRESS_1 = 'john.doe.home@example.com';
 const SECONDARY_ADDRESS_2 = 'john.doe.holidays@example.com';
 const HUGE_ADDRESS = `john${'.doe'.repeat(30)}@example.com`;
 
+/** @type {PrimaryAddressState} */
 const primaryAddress = { state: 'idle', address: PRIMARY_ADDRESS, verified: true };
+/** @type {PrimaryAddressState} */
 const primaryUnverified = { state: 'idle', address: PRIMARY_ADDRESS, verified: false };
+/** @type {Array<SecondaryAddressState>} */
 const secondaryAddresses = [
   { state: 'idle', address: SECONDARY_ADDRESS_1, verified: true },
   { state: 'idle', address: SECONDARY_ADDRESS_2, verified: true },
 ];
 
+const baseItem = {
+  emails: {
+    state: 'loaded',
+    value: {
+      primaryAddress,
+      secondaryAddresses,
+    },
+  },
+};
+
+/**
+ * @typedef {import('./cc-email-list.js').CcEmailList} CcEmailList
+ * @typedef {import('./cc-email-list.types.js').PrimaryAddressState} PrimaryAddressState
+ * @typedef {import('./cc-email-list.types.js').SecondaryAddressState} SecondaryAddressState
+ */
```

Comment:
fix: need to move these after `export default` because it breaks storybook docs  (I don't have a better solution since it's related to Storybook's code)